builtin/providers/aws/resource_aws_ami_copy.go

```diff
@@ -47,6 +47,14 @@ func resourceAwsAmiCopyCreate(d *schema.ResourceData, meta interface{}) error {
 		SourceRegion:  aws.String(d.Get("source_ami_region").(string)),
 	}
 
+	if v, ok := d.GetOk("encypted"); ok {
+		req.Encrypted = aws.Bool(v.(bool))
+	}
```

Comment:
The `optional`, `computed` `schema.TypeBool` and follow up `d.GetOk` seems to bug us down the road. For instance, if you start with `true`, you can't change it to `false`.